examples/miscellaneous/opacitywindow.py

```diff
@@ -0,0 +1,32 @@
+from kivy.app import App
+from kivy.uix.boxlayout import BoxLayout
+from kivy.lang import Builder
+
+Builder.load_string('''
+#:import window kivy.core.window.Window
```

Comment:
Global variable `win` should be available in kv code and if so then this import is not needed and you can use `win` instead of `window` for the rest of the kv code.

Reply:
We cannot use `win` variable. I get an error:
`NameError: name 'win' is not defined`